### ISSUE TITLE
JetBrains IDE error "Module not installed" when importing @trixtateam/phoenix-to-redux

### ISSUE THREAD

**Issue by jeffjenk** [trixtateam/phoenix-to-redux#30]
**Describe the bug**
When importing this module into a JetBrains IDE project the IDE does not seem to be able to find the module. This is problematic as I lose out on IDE assisted cursor context, hints, code completion etc. I'm not sure if this is something that can be fixed in the module itself. I did find some people have had similar problems with other modules starting with `@` (most notably `@vue`) in JetBrains IDEs. 

**To Reproduce**
1. Install `phoenix-to-redux` in project
2. Attempt to import module in JavaScript
3. Observe JetBrains IDE error "Module is not installed"

**Expected behavior**
When installing this module I expect JetBrains IDE to find the module and provide context for available methods etc.

**Screenshots**
![image](https://user-images.githubusercontent.com/936604/182827596-200b2736-d2ae-47a6-88cb-0a9b35bfb837.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - IDE: PhpStorm 2021.2.4 Build #PS-212.5712.51, built on December 22, 2021
 - Version (phoenix-to-js): 4.0.6-beta.0

**Additional context**
If I find a local setup change that fixes the problem I will feed back.

**Comment by jeffjenk** [comment]
Okay I think I found the issue. In the `package.json` the module is being set to: `"module": "dist/@trixtateam/phoenix-to-redux.esm.js"` but when I looked inside the node_modules directory this file is missing:
![image](https://user-images.githubusercontent.com/936604/182851287-fdeb3bf5-2bc9-4f3b-9b85-3df5e3e67a04.png)

I assume its just a build issue or accidental omission. I can do a PR if you like?
